app/commands/solution/queue_head_test_run.rb

```diff
@@ -37,16 +30,28 @@ def handle_latest!
     end
   end
 
+  def should_run_latest?
+    # Always run if we force
+    return true if force
+
+    # Don't run if we're already running this
```

Comment:
```suggestion
    # Don't run if we're already running this
```

Maybe expand a little on what "this" means? In particular, what the important files hash condition does?